packages/connectivity/connectivity/ios/Classes/FLTConnectivityPlugin.m

```diff
@@ -111,12 +74,14 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
     // and the code
     // gets more involved. So for now, this will do.
     result([self statusFromReachability:[Reachability reachabilityForInternetConnection]]);
-  } else if ([call.method isEqualToString:@"wifiName"]) {
+  } else if ([call.method isEqualToString:@"wifiIPAddress"]) {
```

Comment:
Makes sense to move this since it's a very local change

Reply:
sounds good